pkg/crds/names.go

```diff
@@ -235,6 +237,7 @@ var MigratedResources = map[string]bool{
 	"nodepools.management.cattle.io":                                  false,
 	"nodes.management.cattle.io":                                      false,
 	"nodetemplates.management.cattle.io":                              false,
+	"oidcclients.management.cattle.io":                                true,
```

Comment:
Was this intentional?

Reply:
Yes, looking at the diff we're just making this list alphabetical so its easier to find the crd names